snuba/utils/bucket_timer.py

```diff
@@ -111,7 +111,10 @@ def get_projects_exceeding_limit(self) -> List[int]:
                 "project_id: %s, total_processing_time: %s, limit: %s"
                 % (project_id, total_processing_time, self.limit)
             )
-            if total_processing_time > self.limit and len(project_groups) > 1:
+            if total_processing_time > self.limit and (
+                len(project_groups) > 1
+                or get_int_config("s4s_replacements_testing", default=0)
```

Comment:
I think a more descriptive name would be more apt like `allows_skipping_single_project_replacements` I'm also not even sure if this `len(project_groups) > 1` thing even makes sense or not but we can do that later

Reply:
I think it makes sense because if it's the only project that is doing replacements, why not just let it consume all the resources

Reply:
My first thought is:

If a project sent a massive replacements batch and taken up a whole bunch of resources to do it it blocks the queue for any other projects trying to do replacements. Just because this batch only has one project I'm not sure what that says about it taking too long. 

This is just a thought though, I could be misinformed about how the replacer works